fs/copy_linux.go

```diff
@@ -49,22 +50,33 @@ func copyFileContent(dst, src *os.File) error {
 		return errors.Wrap(err, "unable to stat source")
 	}
 
-	n, err := unix.CopyFileRange(int(src.Fd()), nil, int(dst.Fd()), nil, int(st.Size()), 0)
-	if err != nil {
-		if err != unix.ENOSYS && err != unix.EXDEV {
-			return errors.Wrap(err, "copy file range failed")
+	var written int64
+	size := st.Size()
+	first := true
+
+	for written < size {
+		var desired int
+		if size-written > math.MaxInt32 {
```

Comment:
this `size-written` looks ugly without spaces, but apparently it's gofmt way. So I'd suggest to rewrite it as
```go
desired := size - written
if desired > math.MaxInt32 {
	desired=int(math.MaxInt32)
}
```